skt/misc.py

```diff
@@ -24,6 +25,39 @@
 LOGGER = logging.getLogger()
 
 
+@retryable(RuntimeError)
```

Comment:
I'm still not entirely happy with the name: it implies that this function could be retried, but so could any function. I would rather call it `retrying`, or maybe `retrying_on_exception`.